jdbc/src/main/java/org/apache/zeppelin/jdbc/SqlCompleter.java

```diff
@@ -179,7 +179,8 @@ public int complete(String buffer, int cursor, List<InterpreterCompletion> candi
 
 
   private static void fillTableNames(String schema, DatabaseMetaData meta, Set<String> tables) {
-    try (ResultSet tbls = meta.getTables(schema, schema, "%", null)) {
+    try (ResultSet tbls = meta.getTables(schema, schema, "%",
+        new String[]{"TABLE", "VIEW", "ALIAS", "SYNONYM", "GLOBAL TEMPORARY", "LOCAL TEMPORARY"})) {
```

Comment:
I'm afraid this varies depending on the source, for instance it could return quite a different list for SQL server
https://docs.microsoft.com/en-us/sql/connect/jdbc/reference/gettables-method-sqlserverdatabasemetadata